projects/js-packages/script-data/src/types.ts

```diff
@@ -18,6 +18,7 @@ export interface PublicSiteData {
 export interface AdminSiteData {
 	admin_url: string;
 	date_format: string;
+	host: 'woa' | 'atomic' | 'newspack' | 'vip' | 'wpcom' | 'unknown';
```

Comment:
Let us make this optional as it actually is from the backend.

```suggestion
	host?: 'woa' | 'atomic' | 'newspack' | 'vip' | 'wpcom' | 'unknown';
```

Reply:
In practice everything in `AdminSiteData` is optional since it's included in the `JetpackScriptData` interface using `Partial< ... >`.

But perhaps this makes it clearer that this property is even more optional than the other optional properties.

Reply:
Yeah, thanks. I am fine with both.